shared/src/balancer/event_handler.rs

```diff
@@ -45,11 +46,23 @@ pub struct TokensRegistered {
 
 #[derive(Clone, Debug, Eq, PartialEq, Hash)]
 pub struct RegisteredPool {
-    pool_id: H256,
-    pool_address: H160,
-    tokens: Vec<H160>,
-    specialization: PoolSpecialization,
-    block_created: u64,
+    pub pool_id: H256,
+    pub pool_address: H160,
+    pub specialization: PoolSpecialization,
+    pub tokens: Vec<H160>,
+    pub block_created: u64,
+}
+
+impl RegisteredPool {
```

Comment:
Should this be only in #[cfg(test)] ? Otherwise you could also derive Default on RegisteredPool

Reply:
Can't derive default because of specialization and tokens but I'll move it into a test cuz that's where it should be.

Reply:
No longer need `PoolSpecialization` so we can derive Default.